### PR TITLE
Fixes invisible robotic limbs

### LINKED ISSUE #25586: Invisible robotic and augmented limbs
Like the title says, the robotic limbs not including ones made from the fabricator, its only limbs that have been or are attached to people, are completely invisible.  On the floor they don't show up at all but their names do show up if I right click the tile they are on.  On people, it just looks like they're missing the limb, but the vars (unedited) say the robotic limb is there.  This has been going on for me through different commits.  So far this has been for each body part.  In the picture the two invisible limbs are invisible. with their visible counterparts.
invisible limbs

### PR DESCRIPTION
**Pull request by Profakos**
Some methods of obtaining a robotic limb resulted in an invisible appendage, such a through "Modify bodypart" in view variables, or when you became a cyborg. The cause was the lack of subsequent changes to the bodypart's icons, so i have added an extra optional parameter to change_bodypart_status, that updates the icon with a default value.  

I have added a default value for the organic limbs too, just in case.

Fixes #25586

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -127,3 +127,4 @@ Sweaterkittens = Game Master
 Feemjmeem = Game Master
 JStheguy = Game Master
 excessiveuseofcobby = Game Master
+Plizzard = Game Master
```
Comment (TalkingCactus) [inline config/admins.txt]: nice

**Comment by Profakos** [comment]
Darn, i forgot to remove that config file. Not that it matters, as it should be overwritten by server configs, but still...

**Comment by RandomMarine** [comment]
The admins.txt in the repo doesn't apply to the servers. In fact, it's typical for contributors to throw their name in there for convenience with testing.

**Comment by optimumtact** [comment]
@Profakos feel free to leave yourself in as a nod to yourself

**Comment by optimumtact** [comment]
@Profakos can you change those 0 and 1's to be TRUE/FALSE defines since these parameters are technically being used as boolean flags?

Thanks

**Comment by Profakos** [comment]
All done!

**Pull request merged**